apps/plea/models.py

```diff
@@ -484,8 +484,10 @@ def get_court(self, urn, ou_code=None):
 
         if ou_code:
             try:
-                return OUCode.objects.get(ou_code=ou_code[:5]).court
```

Comment:
The OU code (organisational unit) is supplied by Libra for each case and is separate to the URN - it's a Libra field which refers to the court building and room.  We're only interested in the first 5 digits, which refers to the court/building.  As we still don't have strict validation throughout it's still an optional field, with a fall back to matching on the region code (first two digits of URN).